src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -148,6 +148,13 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
       name: this.actionLabels.DELETE,
       disable: (selection: CdTableSelection) => this.getDeleteDisableDesc(selection)
     };
+    const resyncAction: CdTableAction = {
+      permission: 'update',
+      icon: Icons.refresh,
+      click: () => this.resyncRbdModal(),
+      name: this.actionLabels.RESYNC,
```

Comment:
Isn't there any way to add a tooltip here? I think it'd be great to briefly summarize [what this does](https://docs.ceph.com/en/quincy/rbd/rbd-mirroring/#force-image-resync).